jsonschema-validation.xml

```diff
@@ -1176,9 +1179,9 @@
                     the entire resource being described MAY be removed in the future.
                 </t>
                 <t>
-                    When the "deprecated" keyword is applied to an item in an array by means of 
```

Comment:
I think handrews is taking care of whitespace changes already in other cleanup commits.

Reply:
Fixed.

Reply:
Yeah I usually do a formatting-only commit somewhere for this sort of thing.  I'm fine with tacking things you find onto other PRs although they should be done as separate commits.

We should probably write guidance like this down somewhere, but "don't mix formatting changes and content changes in the same commit" is a good rule of thumb.